desktop/keepintouch-100.php

```diff
@@ -0,0 +1,318 @@
+<?php 
+    // This page is hosted in Keyman Desktop Configuration.
+    // Note that it must remain compatible with IE8/9 (yeuch).
+?>
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>Keep in touch with the Keyman team</title>
+  <link href="https://fonts.googleapis.com/css?family=Cabin:400,400italic,500,600,700,700italic|Source+Sans+Pro:400,700,900,600,300|Noto+Serif:400" rel="stylesheet" type="text/css">
+  <script src="//code.jquery.com/jquery-1.11.2.min.js"></script>
```

Comment:
thoughts on copying this to /cdn/dev/js/ ?

Reply:
We could do -- it would come as a later update rather than in this refactor. We will probably want to revisit this page for a future version of Keyman, too.